drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -0,0 +1,673 @@
+if NRF70_RX_WQ_ENABLED
+config NRF70_RX_WQ_PRIORITY
+	int "Priority of the workqueue for handling RX"
+	default 0
+
+config NRF70_RX_WQ_STACK_SIZE
+	int "Stack size of the workqueue for handling RX"
+	default 2048
+endif # NRF70_RX_WQ_ENABLED
+
+if NRF_WIFI_LOW_POWER
+config NRF70_RPU_PS_IDLE_TIMEOUT_MS
+	int "RPU power save idle timeout in milliseconds"
+	default 10
+
+config NRF70_RPU_EXTEND_TWT_SP
+	bool "extending TWT service period"
+	help
+	  In case frames accepted before beginning of SP are not
+	  transmitted before the SP completes then typically they are
+	  dropped to conform to SP window as per specification i.e., no
+	  transmission outside SP window.
+
+	  This feature mitigates the frame loss by transmitting even after SP
+	  completion by using standard contention mechanism which is allowed
+	  in specification but not recommended. As the device is actively transmitting
+	  beyond SP, the power consumption increases depending on the amount
+	  of traffic available at the start of SP.
+
+	  Please note that if a frame is sent after SP starts it will be queued and this
+	  mechanism is not used.
+endif # NRF_WIFI_LOW_POWER
+
+config WIFI_FIXED_MAC_ADDRESS
+	string "WiFi Fixed MAC address in format XX:XX:XX:XX:XX:XX"
+	help
+	  This overrides the MAC address read from OTP. Strictly for testing purposes only.
+
+choice
+	prompt "Wi-Fi MAC address type"
+	default WIFI_OTP_MAC_ADDRESS if WIFI_FIXED_MAC_ADDRESS = ""
+	default WIFI_FIXED_MAC_ADDRESS_ENABLED if WIFI_FIXED_MAC_ADDRESS != ""
+	help
+	  Select the type of MAC address to be used by the Wi-Fi driver
+
+config WIFI_OTP_MAC_ADDRESS
+	bool "Use MAC address from OTP"
+	help
+	  This option uses the MAC address stored in the OTP memory of the nRF70.
+
+config WIFI_FIXED_MAC_ADDRESS_ENABLED
+	bool "fixed MAC address"
+	help
+	  Enable fixed MAC address
+
+config WIFI_RANDOM_MAC_ADDRESS
+	bool "random MAC address generation at runtime"
+	depends on ENTROPY_GENERATOR
+	help
+	  This option enables random MAC address generation at runtime.
+	  The random MAC address is generated using the entropy device random generator.
+
+endchoice
+
+config NRF70_RSSI_STALE_TIMEOUT_MS
+	int "RSSI stale timeout in milliseconds"
+	default 1000
+	help
+	  RSSI stale timeout is the period after which driver queries
+	  RPU to get the RSSI the value.
+	  If data is active (e.g. ping), driver stores the RSSI value from
+	  the received frames and provides this stored information
+	  to wpa_supplicant. In this case a higher value will be suitable
+	  as stored RSSI value at driver will be updated regularly.
+	  If data is not active or after the stale timeout duration,
+	  driver queries the RPU to get the RSSI value
+	  and provides it to wpa_supplicant. The value should be set to lower
+	  value as driver does not store it and requires RPU to provide the
+	  info.
+
+if NETWORKING
+# Finetune defaults for certain system components used by the driver
+config HEAP_MEM_POOL_SIZE
+	default 153000
```

Comment:
Found this seemingly reduntant config given we are using `ADD_SIZE_NRF70`

Reply:
Thanks, removed it, will wait for any other comments before pushing.